src/uct/ib/base/ib_device.h

```diff
@@ -441,4 +443,8 @@ static inline ucs_status_t uct_ib_poll_cq(struct ibv_cq *cq, unsigned *count, st
 
 void uct_ib_handle_async_event(uct_ib_device_t *dev, uct_ib_async_event_t *event);
 
+static inline int uct_ib_device_is_efa(const uct_ib_device_t *dev) {
+    return dev->flags & UCT_IB_DEVICE_FLAG_EFA;
+}
```

Comment:
@yosefe If this is ok, then I can use it instead of `uct_ib_device_has_cq_notify()` to set `has_cq_notify`.

Reply:
IMO, better test it (future proof), than having a hard-coded device id check